requirements.in

```diff
@@ -1,3 +1,5 @@
 gh_issues
 render_engine
 typer
+pip-tools
```

Comment:
I'm -1 on including `pip-tools` in the requirements file here. In general, I think we should be relying on the task defined in our pre-commit config for this.